src/FluentValidation/TestHelper/ITestValidationContinuation.cs

```diff
@@ -9,6 +9,7 @@ public interface ITestValidationWith : ITestValidationContinuation {
 	}
 
 	public interface ITestValidationContinuation : IEnumerable<ValidationFailure> {
+		IEnumerable<ValidationFailure> MatchedFailures { get; }

```

Comment:
This will cause build failures, as adding a new member on a public interface is a binary breaking change outside of a major version release. See comment below for workaround